modules/developers-guide/pages/cli-reference.adoc

```diff
@@ -1375,6 +1395,74 @@ To display the usage information for `+dfx bew+`, run the following command:
 dfx help new
 ----
 
+== dfx identity
+
+Use the `+dfx identity+` command to manage the identities used to communicate with the Internet Computer network. 
+Creating multiple user identities enables you to test user-based access controls.
+
+=== Basic usage
+
+[source,bash]
+----
+dfx identity [subcommand] [flag]
+----
+
+=== Flag
+
+You can use the following optional flags with the `+dfx identity+` command:
+
+[width="100%",cols="<32%,<68%",options="header"]
+|===
+|Flag |Description
+
+|`+-h+`, `+--help+` |Displays usage information.
+
+|`+-V+`, `+--version+` |Displays version information.
+|===
+
+=== Subcommands
+
+Use the following subcommands to specify the operation you want to perform or to view usage information for a specific command.
+
+[width="100%",cols="<32%,<68%",options="header"]
+|===
+|Command |Description
+
+| `+help+` |Displays this usage message or the help of the given subcommand(s).
+
+|`+list+` |Lists existing identities.
+
+|`+new+` |Creates a new identity.
+
+|`+remove+` |Removes an existing identity.
+
+|`+rename+` |Renames an existing identity.
+
+|`+use+` |Specifies the identity to use.
+
+|`+whoami+` |Displays the name of the current identity user context.
+|===
+
+=== Examples
+
+You can use `+dfx identity new+` to create a new user identity named `+ic_admin+` by running the following command:
+
+....
+dfx identity new ic_admin
+....
+
+To rename an identity you have previously created, you can specify the current identity name you want to change **from** and the new name you want to change **to** by running a command similar to the following:
```

Comment:
Do we need to say anything about the `default` identity?  It can be renamed.

Reply:
added to the dfx canister create and dfx identity examples sections